server/controllers/FileController.js

```diff
@@ -0,0 +1,87 @@
+import cloudinary from 'cloudinary';
+import fs from 'fs';
+import del from 'del';
+
+import User from './../models/User';
+
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET,
+});
+
+/**
+ * @class FileController
+ */
+class FileController {
+  /**
+   * Uploads a profile photo
+   *
+   * @static
+   *
+   * @param {Object} request - request object
+   * @param {Object} response - response object
+   *
+   * @memberof FileController
+   *
+   * @returns {void}
+   */
+  static async uploadAvatar(request, response) {
+    try {
+      if (!request.files) {
+        response.status(400).json({
+          error: 'No avatar was uploaded',
+        });
+      } else {
+        const { avatar } = request.files;
+
+        const uploadDir = 'server/uploads';
+
+        if (!fs.existsSync(uploadDir)) {
+          fs.mkdirSync(uploadDir);
+        }
+
+        await avatar.mv(`${uploadDir}/${avatar.name}`);
```

Comment:
A cleaner approach will be to move the code that uploads stuff to a service and call that service here. Your controller is meant to interact with your database and return response to your client. Logic like file uploads or any other kind of logic should be moved out of the controller. So you can create a `CloudinaryService` or `FileUploadService` and put the code for uploading in it.

Reply:
Okay. Thanks. Will do that